src/psalter.php

```diff
@@ -129,14 +133,18 @@ function ($arg) use ($valid_long_options, $valid_short_options) {
 
     --allow-backwards-incompatible-changes=BOOL
         Allow Psalm modify method signatures that could break code outside the project. Defaults to true.
+
 HELP;
 
     exit;
 }
 
-if (!isset($options['issues']) && (!isset($options['plugin']) || $options['plugin'] === false)) {
-    die('Please specify the issues you want to fix with --issues=IssueOne,IssueTwo '
-        . 'or provide a plugin that has its own manipulations with --plugin=path/to/plugin.php' . PHP_EOL);
+if (!isset($options['issues']) &&
+    !isset($options['list-supported-issues']) &&
+    (!isset($options['plugin']) || $options['plugin'] === false)
+) {
+    die('Please specify the issues you want to fix with --issues=IssueOne,IssueTwo or --issues=all, ' .
```

Comment:
Please use `fwrite(STDERR, 'Message');`

Reply:
Done.

Reply:
The `die` is still needed 🙂

Reply:
ah yes of course, thank you. Replaced them with exit(1).